src/test/java/teammates/test/cases/action/InstructorFeedbackResultsDownloadActionTest.java

```diff
@@ -163,6 +167,57 @@ public void testExecuteAndPostProcess() throws Exception {
         expectedFileName = session.getCourseId() + "_" + session.getFeedbackSessionName();
         assertEquals(expectedFileName, result.getFileName());
         verifyFileContentForDownloadWithMissingResponsesHidden(result.getFileContent(), session);
+
+        ______TS("Typical case: results downloadable by question");
```

Comment:
hmm.. I'm trying to be consistent with the previous test cases where we have `Typical case: results downloadable` and `Typical case: results within section downloadable`

Also, it seems that we have always been using either `Typical case` or `Failure case`.